tests/common/features/sea_orm_active_enums.rs

```diff
@@ -53,3 +53,12 @@ pub enum MediaType {
     #[sea_orm(string_value = "3D")]
     _3D,
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, EnumIter, DeriveActiveEnum, DeriveDisplay)]
+#[sea_orm(rs_type = "String", db_type = "Enum", enum_name = "tea")]
+pub enum DisplayTea {
+    #[sea_orm(string_value = "EverydayTea", display_value = "Everyday")]
+    EverydayTea,
+    #[sea_orm(string_value = "BreakfastTea", display_value = "Breakfast")]
+    BreakfastTea,
+}
```

Comment:
Oh wait, actually, did I miss the test case for this? There are no corresponding asserts?

As this is the only place the `display_value` attribute is used.

Reply:
Here is the test case.
https://github.com/SeaQL/sea-orm/pull/1726/files#diff-a85e90f626db076a34bb42800eafe047a773a6ef5fe19d05f3a284efb3c16f3aR802-R808